presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -155,7 +156,14 @@ else if (element instanceof LikeClause) {
 
         ConnectorTableMetadata tableMetadata = new ConnectorTableMetadata(tableName.asSchemaTableName(), ImmutableList.copyOf(columns.values()), finalProperties, statement.getComment());
 
-        metadata.createTable(session, tableName.getCatalogName(), tableMetadata);
+        try {
```

Comment:
Use equals instead of == and to make it more readable we can do

```
        catch (PrestoException e) {
            if (ALREADY_EXISTS.toErrorCode().equals(e.getErrorCode())) {
                if (!statement.isNotExists()) {
                    throw e;
                }
            }
            else {
                throw e;
            }
        }
```